kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -104,11 +106,11 @@
         if (this.isInFullscreen) {
           return {
             position: 'absolute',
-            top: '50px',
+            top: '37px',
             bottom: 0,
           };
         }
-        return { height: '560px' };
+        return { height: this.iframeHeight };
```

Comment:
I don't know if this matters, so much, but making the container height = iframe height,  means the container is smaller than it needs to be. (see that the container is highlighted blue below)

![image](https://user-images.githubusercontent.com/10248067/86292448-a1657400-bba5-11ea-86f3-d9d85d615b0b.png)

To make the container exactly the right height, I think you need to add the extra pixels to account for the Fullscreen toolbar.

Reply:
> I don't know if this matters, so much, but making the container height = iframe height, means the container is smaller than it needs to be. (see that the container is highlighted blue below)
> 
> ![image](https://user-images.githubusercontent.com/10248067/86292448-a1657400-bba5-11ea-86f3-d9d85d615b0b.png)
> 
> To make the container exactly the right height, I think you need to add the extra pixels to account for the Fullscreen toolbar.

This is fixed now as well.  There's now a central point of control for setting the default content height and the topbar height.